lib/hard_data.dart

```diff
@@ -28,12 +28,14 @@ const List<ReusableCard> staffWelfare = [
     description: '3 guides',
     color: Colors.white,
     routeTo: Routes.ppe,
+    margin: EdgeInsets.all(0.0),
```

Comment:
Sorry I don't think putting layout/ui into the data is a good idea. A better approach would be to give these a diff type (eg make a subclass Reuseablecard) and then in UI apply the different margins/heights based on the data type

Reply:
That's true, so then even the colour should not be there.  I went for a quick fix without impacting any other view that uses the ReusableCard class. I'll follow your approach.

Reply:
Gonna have to disagree with @maks here. `ReusableCard` is a **widget**, not a data structure. It already contains fields for `height, elevation, color, fallback` onClick functions.
Because it already contained the color field, there I see no issue in assigning it here. Certainly, that makes more sense to me than creating new Blue, Green, and White Widget classes whose only difference is the colour.

@maks is trying to make it a data class. Ideally, it should be a pure data object, but that would have significant impacts, in particular for how the `fallback` handler is wired up.

Reply:
@rockgecko-development yes I only noticed that after I talked about this directly with @alessandrofavero on slack and had look at code in editor vs the diffs here.
So definitely this needs to be fixed, we have proper data model classes now so there should not be any Widgets in the data layer.
As discusses on slack with @alessandrofavero I'm fine for this PR to go in just fixing the immediate layout tweaks and Im going to follow up with PR to fix the structural issues.